techniques/html/H99.html

```diff
@@ -51,64 +51,64 @@
 
 <div class="default-grid with-gap leftcol">
 <aside class="box nav-hack sidebar standalone-resource__sidebar">
-    <header class="box-h">Page Contents</header>
+    <header class="box-h">このページの内容</header>
     <div class="box-i">
       <nav aria-label="page contents" class="navtoc">
-        <ul><li><a href="#technique">About this Technique</a></li>
-<li><a href="#description">Description</a></li>
-<li><a href="#examples">Examples</a></li>
-<li><a href="#resources">Related Resources</a></li>
-<li><a href="#tests">Tests</a></li>
+        <ul><li><a href="#technique">このテクニックについて</a></li>
+<li><a href="#description">解説</a></li>
+<li><a href="#examples">事例</a></li>
+<li><a href="#resources">関連リソース</a></li>
+<li><a href="#tests">検証</a></li>
 </ul>
       </nav>
     </div>
 </aside>
 
 <main id="main" class="standalone-resource__main">
 		
-<h1><span>Technique H99:</span>Provide a page-selection mechanism</h1>
+<h1><span>テクニック H99:</span>ページ選択メカニズムを提供する</h1>
 
 
 
 
 <section id="technique" class="box">
-	<h2 class="box-h box-h-icon">About this Technique</h2>
+	<h2 class="box-h box-h-icon">このテクニックについて</h2>
 	<div class="box-i">
 
 
-  <p>This technique is not referenced from any Understanding document.</p>
+  <p>このテクニックは、どの解説書からも参照されていない。</p>
 
 
 
-  <p>This technique applies to digital publications derived from a fixed layout source or that include static page break locations to enable common points of reference.</p>
+  <p>このテクニックは、固定レイアウトのソースから派生したデジタル出版物、又は共通の参照ポイントを提供するために静的な改ページ位置を含むデジタル出版物に適用される。</p>
 
-<p>User agents may provide a specialized interface to the page list rather than render it directly, especially in ebook formats. Users might be presented with an input box into which to enter the page number they wish to go to, for example.</p></div>
+<p>ユーザエージェントは、特に電子書籍形式の場合、ページリストを直接レンダリングするのではなく、専用のインタフェースを提供する場合がある。例えば、利用者が移動したいページ番号を入力するための入力ボックスが提示される場合がある。</p></div>
 </section>
 	
 
 
 		
 		<section id="description">
-			<h2>Description</h2>
+			<h2>解説</h2>
 
-			<p>The objective of this technique is to ensure that a page-selection mechanism is provided that makes available programmatically all set page locations in the digital publication.</p>
+			<p>このテクニックの目的は、デジタル出版物内の全ての設定されたページ位置をプログラムによる利用が可能にするページ選択メカニズムを確実に提供することである。</p>
 
-			<p>The page-selection mechanism provides a way to locate static page break locations (i.e., where each new page begins) within a digital publication. These locations typically come from a print equivalent of the edition and allow synchronization in, for example, a mix of print and digital publications in environments such as schools and universities.</p>
+			<p>ページ選択メカニズムは、デジタル出版物内の静的な改ページ位置 (つまり、それぞれの新しいページの開始位置) を特定する方法を提供する。この位置は通常、印刷版と同等な版に由来し、例えば学校や大学などの環境において、印刷版とデジタル版が混在する環境で同期を可能にする。</p>
```

Comment:
「環境」が重複しているので、訳を調整してみました。「in a mix of print and digital publications」の「in」を「〜する場合の」としてみましたが、、

```suggestion
			<p>ページ選択メカニズムは、デジタル出版物内の静的な改ページ位置 (つまり、それぞれの新しいページの開始位置) を特定する方法を提供する。この位置は通常、印刷版と同等な版に由来し、例えば学校や大学などの環境において、印刷版とデジタル版が混在する場合の同期を可能にする。</p>
```